rdkit/Chem/AllChem.py

```diff
@@ -438,6 +439,65 @@ def AssignBondOrdersFromTemplate(refmol, mol):
     return mol2
 
 
+def GetNumOfUnsaturatedBonds(atom):
+    """
+    Returns
+        The number of electrons an atom is using for pi bonding
+        OR
+        The Number of Unsaturated bonds, an atom has
+    
+    >>> m = Chem.MolFromSmiles('C=C')
+    >>> GetNumOfUnsaturatedBonds(m.GetAtomWithIdx(0))
+    1
+    
+    >>> m = Chem.MolFromSmiles('C#CC')
+    >>> GetNumOfUnsaturatedBonds(m.GetAtomWithIdx(0))
+    2
+    
+    >>> GetNumOfUnsaturatedBonds(m.GetAtomWithIdx(1))
+    2
+    
+    >>> m = Chem.MolFromSmiles('O=C=CC')
+    >>> GetNumOfUnsaturatedBonds(m.GetAtomWithIdx(0))
+    1
+    
+    >>> GetNumOfUnsaturatedBonds(m.GetAtomWithIdx(1))
+    2
+    
+    >>> GetNumOfUnsaturatedBonds(m.GetAtomWithIdx(2))
+    1
+    
+    >>> GetNumOfUnsaturatedBonds(m.GetAtomWithIdx(3))
+    0
+    
+    >>> m = Chem.MolFromSmiles('c1ccccc1')
+    >>> GetNumOfUnsaturatedBonds(m.GetAtomWithIdx(0))
+    1
+    
+    >>> m = Chem.MolFromSmiles('S(=O)(=O)(O)O')
```

Comment:
Perhaps add, just before these last two examples, something like:

For non-planar environments we return the number of multiple bonds that the atom is involved in (with triple bonds counting twice).